i18n/zh-CN/docusaurus-plugin-content-docs/current/data_node/load_node/elasticsearch.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Elasticsearch
+sidebar_position: 8
+---
+
+## Elasticsearch Load 节点
+
+Elasticsearch Load 节点允许将数据写入到 Elasticsearch 引擎的索引中。本文档描述运行 SQL 查询时如何设置 Elasticsearch Load 节点。
+
+连接器可以工作在 Upsert 模式，使用 DDL 中定义的主键与外部系统交换 UPDATE/DELETE 消息。
+
+如果 DDL 中没有定义主键，那么连接器只能工作在 Append 模式，只能与外部系统交换 INSERT 消息。
+
+## 支持的版本
+
+| Load 节点                             | 版本                                                 | 
+|-------------------------------------|----------------------------------------------------|
+| [elasticsearch](./elasticsearch.md) | [ElasticSearch](https://www.elastic.co/): 6.x, 7.x |
```

Comment:
The `ElasticSearch` should be `Elasticsearch`.

Reply:
done